content/en/network_monitoring/cloud_network_monitoring/network_analytics.md

```diff
@@ -143,22 +143,80 @@ Neutral tags are tags that are not specific to a client or server, and instead a
 
 The following is the list of neutral tags available for use:
 
-| Tag                     | Description                                                                                   |
-|-------------------------|-----------------------------------------------------------------------------------------------|
-| `is_agent_traffic`      | Indicates if the traffic was generated by the Datadog Agent.                                  |
-| `tls_encrypted`         | Specifies if the connection is encrypted using TLS.                                           |
-| `tls_cipher_suite`      | Identifies the TLS cipher suite used (for example, `tls_ecdhe_rsa_with_aes_128_gcm_sha256`).          |
-| `tls_cipher_insecure`   | Indicates if the cipher used is considered secure.                                            |
-| `tls_version`           | The TLS version used (`tls_1.2` or `tls_1.3`).                                                |
-| `tls_client_version`    | The TLS versions supported by the client (`tls_1.2` or `tls_1.3`).                            |
-| `gateway_id`            | Unique identifier for the AWS gateway resource.                                               |
-| `gateway_type`          | Type of AWS gateway (Internet, NAT, or Transit).                                              |
-| `gateway_region`        | AWS region of the gateway (for example, `us-east-1`).                                                |
-| `gateway_availability-zone` | Availability zone hosting the gateway (for example, `us-east-1a`).                                |
-| `gateway_public_ip`     | Public IP address assigned to the NAT gateway.                                                |
-| `tgw_attachment_id`     | Unique identifier for the AWS Transit Gateway attachment.                                     |
-| `tgw_attachment_type`   | Type of Transit Gateway attachment (for example, VPC, VPN, Direct Connect).                          |
-| `vpc_endpoint_id`       | Unique identifier for the VPC endpoint.                                                       |
+<table>
+<thead>
+<tr>
+<th style="white-space: nowrap; width: 300px; min-width: 300px;">Tag</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td><code>gateway_availability-zone</code></td>
```

Comment:
`intra_availability_zone` uses an underscore, so I wanted to double check here - should this be a hyphen or an underscore at the end?